examples/express-sample/routes/index.js

```diff
@@ -9,6 +9,23 @@ router.get("/", function (req, res, next) {
   });
 });
 
+router.get("/skip", function (req, res, next) {
+  raygunClient
+    .send("Error to skip")
+    .then((message) => {
+      res.render("send", {
+        title: "Skip sending message",
+        body: `Message should be null: ${message}`,
+      });
+    })
+    .catch((error) => {
+      res.render("send", {
+        title: "Failed to send error to Raygun",
```

Comment:
Should this say  -
```suggestion
        title: "Failed to skip sending error to Raygun",
```

Reply:
I don't think that would be correct, failing to skip would also mean that it was sent by mistake. Maybe just "There was an error" would be more accurate.

Reply:
Same as above. I think I read this wrong! Will resolve this thread